config/robots.rb

```diff
@@ -2799,4 +2799,86 @@
         }
       }
     )
+
+    # Bioscan robots
+
+    custom_robot(
+      'beckman-lilys-96-stock-preparation',
+      name: 'Beckman LILYS-96 Stock Preparation',
+      beds: {
+        bed(9).barcode => {
+          purpose: 'LILYS-96 Stock',
+          states: ['passed'],
+          label: 'Bed 9',
+          target_state: 'passed'
+        }
+      }
+    )
+
+    custom_robot(
+      'beckman-lilys-96-stock-to-lbsn-96-lysate',
+      name: 'Beckman LILYS-96 Stock => LBSN-96 Lysate',
+      verify_robot: true,
+      beds: {
+        # TODO: confirm beds
```

Comment:
Does this need a backlog item adding and a reference to it putting here?  Bare TODOs live forever in the code and don't get done.

Reply:
remove comment